rest/config_legacy.go

```diff
@@ -236,6 +251,62 @@ func (lc *LegacyServerConfig) ToStartupConfig() (*StartupConfig, DbConfigMap, er
 	return &sc, dbs, nil
 }
 
+func gocbv1tov2AddressUpgrade(server string) (newServer, username, password string, err error) {
```

Comment:
I think this is better described as legacyServerAddressUpgrade, with a comment added about what work it's doing.